### PR TITLE
System integration tests: Set Genesis block based on the nodes

### PR DESCRIPTION
**Pull request by Geod24**
As a client, we don't know what is the "correct" genesis block at the moment.
The best option so far is to call `setGenesisBlock` using node-provided data.

### CODE REVIEW

**Review by AndrejMitrovic — APPROVED** [review]

File: tests/system/source/main.d [hunk 1]
```diff
@@ -80,6 +85,7 @@ int main (string[] args)
 
         checkBlockHeight(addresses, 1);
     }
+    // Warning: GenesisBlock() is invalid here
```
Comment (linked0) [inline tests/system/source/main.d]: Could you explain the reason why the GenesisBlock is invalid in this line, 88?

**Review by linked0 — APPROVED** [review]
LGTM

**Comment by AndrejMitrovic** [comment]
I think if you rebase this it will go green.

File: tests/system/source/main.d (hunk 1 above)
Comment (Geod24) [reply tests/system/source/main.d]: Sure! I expanded the description.

**Comment by Geod24** [comment]
Good thing is that this is not block #1005 anymore. But it's still not passing for unknown reasons.

**Comment by Geod24** [comment]
Green :O

**Pull request merged**